project2-HexColorsGradient/src/Button.js

```diff
@@ -0,0 +1,24 @@
+export default function Button({ $target, onClick }) {
+  const $button = document.createElement("button");
+  $target.append($button);
+  $button.textContent = "Click Me!";
+
+  $button.addEventListener("click", (e) => {
+    const generateRandomDirection = () => {
+      return Math.floor(Math.random() * 2) === 0 ? `right` : `left`;
+    };
```

Comment:
return 만 있는 함수라면 이렇게 줄여쓸 수 있습니다!
```suggestion
    const generateRandomDirection = () =>  Math.floor(Math.random() * 2) === 0 ? `right` : `left`;
```

Reply:
이렇게 써주는 게 훨씬 깔끔하겠네요! 감사합니다, 정희님!